tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -25,19 +32,52 @@ export const emptySessionDataForRadioSelect: Session = {
 
 export const nhsAppRadioSelectedSessionData: Session = {
   __typename: 'SessionStorage',
-  id: '3d90000-6666-0000-0000-95eb27590002',
+  id: 'nhsapp-6666-0000-0000-95eb27590002',
   createdAt: '2024-09-19T23:36:20.815Z',
   updatedAt: '2024-09-19T23:36:20.815Z',
   templateType: TemplateType.NHS_APP,
   nhsAppTemplateName: '',
   nhsAppTemplateMessage: '',
 };
 
+export const emailRadioSelectedSessionData: Session = {
```

Comment:
To reduce some of the noise of these sessions you could make use of the `SessionFactory`. Not a blocker.